ai2thor/tests/test_unity.py

```diff
@@ -1598,3 +1598,22 @@ def test_unsupported_manipulathor(controller):
     )
     event = controller.step(action="PickupObject", objectId=objectId, forceAction=True)
     assert not event, "PickupObject(objectId) should have failed with agentMode=arm"
+
+
+@pytest.mark.parametrize("controller", fifo_wsgi)
+def test_invalid_arguments(controller):
+    controller.reset()
+    with pytest.raises(ValueError):
+        event = controller.step(
+            action="PutObject",
+            x=0.0,
+            y=0.0,
+            z=1.0,
+            forceAction=False,
+            placeStationary=True
+        )
+        print("Error code {0}".format(event.metadata["errorCode"]))
+        assert not event.metadata["lastActionSuccess"], "Extra parameter 'z' in action"
+        assert event.metadata[
+            "errorMessage"
+        ], "errorMessage with invalid argument"
```

Comment:
I'm a bit confused about the semantics of `pytest.raises(ValueError)`. Here we should expect that this call results in a `ValueError` being raised, correct? Shouldn't this mean that the asserts won't be hit?

Reply:
Good point, this meant to be escaped left, now fixed